telco-hub/configuration/reference-crs/required/acm/acmMCO.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: observability.open-cluster-management.io/v1beta2
+kind: MultiClusterObservability
+metadata:
+  name: observability
+  annotations:
+    mco-disable-alerting: "true"
+spec:
+  advanced:
+    retentionConfig:
+      blockDuration: 2h
+      deleteDelay: 48h
+      retentionInLocal: 3d
+      retentionResolutionRaw: 3d
+  enableDownsampling: false
+  observabilityAddonSpec:
+    enableMetrics: true
+    interval: 300
+  storageConfig:
+    alertmanagerStorageSize: 10Gi
+    compactStorageSize: 100Gi
+    metricObjectStorage:
+      key: thanos.yaml
+      name: thanos-object-storage
+    receiveStorageSize: 25Gi
+    ruleStorageSize: 10Gi
+    storeStorageSize: 25Gi
+
```

Comment:
Is an explicit config of storageClass needed?

Reply:
no, because it uses an S3 connection that has to be created as a secret. 
I did not include the secret, because I consider this as part of the installation. My intention here, by the moment, is to create basic objects/fields for validation.